docs/dev/libvirt/README.md

```diff
@@ -151,6 +151,30 @@ $ sudo firewall-cmd --get-active-zones
 FedoraWorkstation
   interfaces: enp0s25 tun0
 ```
+
+*NOTE:* The installer creates a libvirt network and a `tt0` bridge. It is
```

Comment:
Installer does seem to be creating it, rather it is using it...

Reply:
Unless I'm missing something, it does, through Terraform: https://github.com/openshift/installer/blob/master/data/data/libvirt/main.tf#L33